packages/test/local-server-stress-tests/src/test/localServerStressOrderSequentially.spec.ts

```diff
@@ -103,7 +103,7 @@ describe("Local Server Stress with rollback", () => {
 			...[12], // Values differ at key
 			...[28, 30], // Key not found or value not matching key
 			...[15, 31, 38], // Number of keys not same (directory)
-			...[24], // have different number of keys (map)
+			...[], // have different number of keys (map)
```

Comment:
this line should be deleted

Reply:
Is that true for the other empty ones below (108/109)?  Wondering if there's a convention here I'm missing

Reply:
Done in latest